packages/components/src/icon/index.js

```diff
@@ -8,24 +8,24 @@ import { cloneElement, createElement, Component, isValidElement } from '@wordpre
  */
```

Comment:
Hi @tyxla @youknowriad 👋 As far as I can see you are the authors of `Icon` component. Could you be able to review our changes? Thanks

Reply:
Hey @pinarol! I'm no longer working on Gutenberg for a while now, so I'd defer to someone from @WordPress/gutenberg-core to review this one.